spec/controllers/sign_up/recovery_codes_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe SignUp::RecoveryCodesController do
+  describe '#update' do
+    it 'redirects to the profile page' do
+      stub_sign_in
+      subject.current_user.recovery_code = 'foo'
+
+      patch :update
+
+      expect(response).to redirect_to profile_path
+    end
+  end
+end
```

Comment:
do you think we should do that in lieu of the feature spec I added? I started with that since I was going outside on with specs, and so controller spec felt like it would be double coverage. If we wanted, we could move the feature spec to a controller spec but having both feels like overkill to me.